nautobot/core/settings.py

```diff
@@ -4,7 +4,7 @@
 from django.contrib.messages import constants as messages
 
 from nautobot import __version__
-from nautobot.core.settings_funcs import is_truthy, parse_redis_connection
+from nautobot.core.settings_funcs import parse_redis_connection
```

Comment:
Might need to keep this one - not sure if we have places or plugins that are doing `from nautobot.core.settings import is_truthy`, which this would break.